source/product-stories.erb

```diff
@@ -89,7 +89,7 @@ product_stories:
         <%= image_tag product_story.image, alt: product_story.client, class: "#{class_name}__image", srcset: retina_srcset(product_story.image) %>
         <div class="<%= class_name %>__content">
           <div class="<%= class_name %>__client"><%= product_story.client %></div>
-          <div class="<%= class_name %>__title"><%= product_story.title %></div>
+          <div class="<%= class_name %>__paragraph"><%= product_story.title %></div>
```

Comment:
Why did we rename this to `paragraph`? I agree that calling the content `title` isn't accurate, but paragraph is a bit generic to my mind. How about `description`? Whichever name we use I suggest that the class name and the key match in the same way as we've done for `client`.